src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php

```diff
@@ -187,7 +204,7 @@ public function apply($text)
         $unsetCodes = [];
 
         if (null === $this->handlesHrefGracefully) {
-            $this->handlesHrefGracefully = 'JetBrains-JediTerm' !== getenv('TERMINAL_EMULATOR');
+            $this->handlesHrefGracefully = $this->checkHrefSupport();
```

Comment:
not sure about the added method, doing this inline LGTM :)
`$this->handlesHrefGracefully = 'JetBrains-JediTerm' !== getenv('TERMINAL_EMULATOR') && !getenv('KONSOLE_VERSION');`